patches/sandbox-policy-win-sandbox_win.cc.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/sandbox/policy/win/sandbox_win.cc b/sandbox/policy/win/sandbox_win.cc
-index d63d942c7e3e894884403d720e29b06f882a5905..506f57ca346749abe18fa3800486cb8a63ed7b06 100644
+index d63d942c7e3e894884403d720e29b06f882a5905..ba3c42b6b0f6edcdec8ee458bb3f77d0fe3c8f23 100644
 --- a/sandbox/policy/win/sandbox_win.cc
 +++ b/sandbox/policy/win/sandbox_win.cc
 @@ -887,6 +887,7 @@ ResultCode SandboxWin::StartSandboxedProcess(
        launcher_process_command_line.HasSwitch(switches::kNoSandbox)) {
      base::LaunchOptions options;
      options.handles_to_inherit = handles_to_inherit;
-+    BraveLaunchOption(cmd_line, &options);
++    BraveLaunchOption(process_type, &options);
```

Comment:
can we convert this to a define since you're changing it anyway?

Reply:
sure, will do

Reply:
fixed